libs/dapp-config/src/anchors/anchor-config.ts

```diff
@@ -1,38 +1,54 @@
-import { CurrencyRole, EVMChainId } from '@webb-tools/dapp-types';
-import { ChainType, calculateTypedChainId } from '@webb-tools/sdk-core';
+import { EVMChainId } from '@webb-tools/dapp-types';
+import { calculateTypedChainId, ChainType } from '@webb-tools/sdk-core';
 
-import { ApiConfig } from '../api-config';
-import { CurrencyConfig } from '../currencies';
+// 0xa1a2b7e08793b3033122b83cbee56726678588b5 - webbWETH - mocked backend
+// 0x38e7aa90c77f86747fab355eecaa0c2e4c3a463d - webbAlpha - mocked backend
+// 0x9678647b9fcb0039652a16dba688bd067d6e5077 - webbStandalone - dkg backend
```

Comment:
We can remove reference to `0x9678647b9fcb0039652a16dba688bd067d6e5077 - webbStandalone - dkg backend` in favour of `0xf8c9d24e3bc3e2d3eddde507079b08e82f239fc4 - webbtTNT-standalone`

Reply:
No need to support both